rust/arrow/src/array.rs

```diff
@@ -784,6 +996,76 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_date64_array_from_vec_option() {
+        // Test building a primitive array with null values
+        // we use Int32 and Int64 as a backing array, so all Int32 and Int64 convensions work
```

Comment:
typo in comments, should be `conversions` I think...  (repeated below)

Reply:
I meant `conventions`, thanks, I'll fix the typos